runelite-api/src/main/java/net/runelite/api/AnimationID.java

```diff
@@ -59,6 +59,31 @@ public final class AnimationID
 	public static final int WOODCUTTING_2H_CRYSTAL = 10072;
 	public static final int WOODCUTTING_2H_CRYSTAL_INACTIVE = 10073;
 	public static final int WOODCUTTING_2H_3A = 10074;
+	/*Ent animation shared with Canoe cutting*/
+	public static final int WOODCUTTING_ENT_BRONZE = 3291;
+	public static final int WOODCUTTING_ENT_IRON = 3290;
+	public static final int WOODCUTTING_ENT_STEEL = 3289;
+	public static final int WOODCUTTING_ENT_BLACK = 3288;
+	public static final int WOODCUTTING_ENT_MITHRIL = 3287;
+	public static final int WOODCUTTING_ENT_ADAMANT = 3286;
+	public static final int WOODCUTTING_ENT_RUNE = 3285;
+	public static final int WOODCUTTING_ENT_DRAGON = 3292;
+	public static final int WOODCUTTING_ENT_CRYSTAL = 8327;
```

Comment:
Do you have a crystal axe to confirm which one is active versus inactive, or is this a guess? Visually, I think this is backwards, as 8327 is closer to white (which would match the look of the inactive axe) while 8323 is more blue.

Reply:
I can confirm crystal is 8323 instead of 8327